### PR TITLE
Dist/august 2022 disburse19

### PR DESCRIPTION
**Pull request by luxumbra**
I managed to leave the files from the last mint staged in my local repo until this eve when I came to do deduct the seeds from the ledger to prepare for the next mint. Of course there's merge conflicts with `ledger.json`. I ran the `rebase-ledger.js` script as mentioned in the readme and that seemed to complete fine but looks like there's still an issue. 

Some assistance to resolve this would be appreciated. 🙏🏻

### CODE REVIEW

**Comment by alalonde** [comment]
OK I did a merge based on the commit timestamps. I didn't check out the whole repo or load data so I couldn't check to see if the ledger was still valid though

File: data/ledger.json [hunk 1]
```diff
@@ -13775,11 +13775,29 @@
...
+=======
+>>>>>>> 1259828 (rebased ledger as per docs after I negelected to commit the last calc)
```
Comment (polimyl) [inline data/ledger.json:13787]: :eyes:

**Comment by polimyl** [comment]
also the 25/75 budget was applied to both last and current distro, if thats intentional disregard

**Comment by luxumbra** [comment]
> also the 25/75 budget was applied to both last and current distro, if thats intentional disregard

This is the last distro which happened mid september. The previous one was jun/july. The values in grain.json can be seen here: 051ac413c2fea29476d355f2aba120a0809f7633.

Once this PR is merged, I can then do the seeds deduction from the ledger and then the present distribution which will be mid-sept to *whatever date in October* :)

File: data/ledger.json (hunk 1 above)
Comment (alalonde) [reply data/ledger.json:13787]: Whoops, missed that one

Comment (alalonde) [reply data/ledger.json:13787]: No biggie, just editing the "ledger" 😅

Comment (luxumbra) [reply data/ledger.json:13787]: > No biggie, just editing the "ledger" sweat_smile

what could possibly go rong? 😄 I appreciate your intervention & 👀  @alalonde & @polimyl 🙏🏻

**Comment by luxumbra** [comment]
@alalonde is this cool to merge now?

**Pull request merged**